presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -147,6 +147,7 @@
     private static final String WILDCARD_EXPRESSION = "";
     private static final int BATCH_GET_PARTITION_MAX_PAGE_SIZE = 1000;
     private static final int BATCH_CREATE_PARTITION_MAX_PAGE_SIZE = 100;
+    private static final int AWS_GLUE_GET_PARTITIONS_MAX_RESULTS = 128;
```

Comment:
why 128 and not some other value (e.g larger)?

Reply:
Ultimately we need to find a balance between how much data you transfer at once vs how many iterations do you need on average for a typical use case. 128 is much larger than the current default (1) so I would start with this. I also think that we could make this configurable. What would you like to see as the default value instead?

Reply:
Is there some value that @l1x recommends?

Reply:
I guess 128.

Reply:
haha. I meant what AWS recommends?